python/src/main.cpp

```diff
@@ -2,13 +2,22 @@
 #include <pybind11/stl.h>
 #include <pybind11/numpy.h>
 
-#include <kompute/Kompute.hpp>
+#include <AggregateHeaders.cpp>
```

Comment:
Oh just noticed here you added the aggregate headers - I don't think this file is exported as part of the package, this would have to be reverted to `kompute/Kompute.hpp` - does this re-introduce the bug?